daprdocs/content/en/operations/components/setup-pubsub/supported-pubsub/setup-azure-eventhubs.md

```diff
@@ -5,51 +5,63 @@ linkTitle: "Azure Events Hub"
 description: "Detailed documentation on the Azure Event Hubs pubsub component"
 ---
 
-## Setup Azure Event Hubs
-
-Follow the instructions [here](https://docs.microsoft.com/en-us/azure/event-hubs/event-hubs-create) on setting up Azure Event Hubs.
-Since this implementation uses the Event Processor Host, you will also need an [Azure Storage Account](https://docs.microsoft.com/en-us/azure/storage/common/storage-account-create?tabs=azure-portal).
-
-## Create a Dapr component
-
-The next step is to create a Dapr component for Azure Event Hubs.
-
-Create the following YAML file named `eventhubs.yaml`:
+## Format
```

Comment:
See note on Kafka. I'd rename this "Component format" and consolidate "Apply the configuration" and "Create a Dapr component" under it.